client/reader/stream/post-lifecycle.jsx

```diff
@@ -27,7 +27,7 @@ import EmptySearchRecommendedPost from './empty-search-recommended-post';
 
 const ConnectedCombinedCard = fluxPostAdapter( CombinedCard );
 
-export default class PostLifecycle extends React.PureComponent {
```

Comment:
why's this changed?

Reply:
in React 16 you get errors for extending PureComponent but also overriding `shouldComponentUpdate`.  Its one or the other

Reply:
they manifest themselves as annoying warnings in console.  wouldn't actually break anything